packages/synapse-react-client/src/components/ChallengeDetailPage/ChallengeDetailPage.tsx

```diff
@@ -73,37 +92,29 @@ export function ChallengeDetailPage({ projectId }: ChallengeDetailPageProps) {
         onJoinClick={onJoinClick}
         onLeaveClick={onLeaveClick}
       />
-      {showRequirements && (
-        <AccessRequirementList
-          entityId={projectId}
-          teamId={challenge?.participantTeamId}
-          onHide={handleRequirementsHide}
-          renderAsModal={true}
-          requestObjectName={project?.name}
-          dialogTitle="Challenge Terms and Conditions"
-        />
-      )}
-      {showWizard && (
-        <ChallengeTeamWizard
-          projectId={projectId}
-          onClose={() => {
-            toggleShowWizard(false)
-          }}
-          isShowingModal={true}
-        />
-      )}
-      {showLeaveConfirm && (
-        <ConfirmationDialog
-          open={showLeaveConfirm}
-          title={'Leave this Challenge?'}
-          content={'Are you sure you want to leave this Challenge?'}
-          onCancel={() => setShowLeaveConfirm(false)}
-          onConfirm={() => {
-            doLeaveChallenge()
-            setShowLeaveConfirm(false)
-          }}
-        />
-      )}
+      <ChallengeRequirementsModal
+        open={showRegistrationModal}
+        projectId={projectId}
+        onRegisterComplete={handleRegistrationComplete}
+        onCancel={handleRequirementsHide}
+      />
```

Comment:
Use a new component that 
1. Shows the ARs
2. Blocks registration until the ARs are complete
3. When the user has completed all ARs, the user can click "Register" and add themselves to the participant team

Reply:
@vpchung : FYI.  I think these are all known constraints of the Challenge portal that were highlighted during the design, but if any of this is new information then please let us know (and we'll open up tickets to track)

Reply:
> It is not possible to create/join a submission team other than when you register for a challenge.

Does this mean that as team leader, I will not be able to invite members after the registration screen?  Or can I still invite members to join if I were to go to the team page on Synapse?  Likewise, as a team member, can I bypass this screen until a team leader has created a submission team, and then wait for their invite?

Reply:
> Only 'open' participant teams are supported. You cannot request access to a participant team that requires team manager approval

I want to say this is OK (for now?).  I have yet to organize a challenge where manager approval is needed first before joining the participants team.

Reply:
> It is not possible to see access requirements after joining the participant team without leaving the team and rejoining.

No worries, the ARs are (should be) posted on the challenge site as well, under the **Data Conditions for Use**.

Reply:
@vpchung 
> Does this mean that as team leader, I will not be able to invite members after the registration screen? Or can I still invite members to join if I were to go to the team page on Synapse? 

Correct, but I don't think we guide users to go to Synapse, so it probably won't be clear how to invite more people to their team for most users.

> Likewise, as a team member, can I bypass this screen until a team leader has created a submission team, and then wait for their invite?

Creating a submission team is optional, but IMO the UI doesn't really make it clear that it is a separate, optional step. The user can close the modal or close the tab. 

Being able to act on a team invitation within the challenge portal is tracked in PORTALS-2940.